accounts/views.py

```diff
@@ -37,3 +38,8 @@ def form_valid(self, form):
         super(RegisterView, self).form_valid(form)
         messages.success(self.request, 'Verification link sent! Please check your email.')
         return redirect(self.success_url)
+
```

Comment:
Remove these empty blank lines at the end of the file. According to PEP8, each file should have just a single blank line at the end.